container-toolkit/release-notes.md

```diff
@@ -6,9 +6,36 @@
 
 # Release Notes
 
-This document describes the new features, improvements, fixed and known issues for the NVIDIA Container Toolkit.
+This document describes the new features, improvements, fixes and known issues for the NVIDIA Container Toolkit.
 
-______________________________________________________________________
+## NVIDIA Container Toolkit 1.17.8
+
+This release of the NVIDIA Container Toolkit `v1.17.8` is a bugfix release.
+
+### Fixes and Features
+
+- Updated the ordering of Mounts in CDI to have a deterministic output. This makes testing more consistent.
+- Added NVIDIA_CTK_DEBUG envvar to hooks.
+
+#### Enhancements to libnvidia-container
+
+- Fixed a bug in setting the default for the `--cuda-compat-mode` flag. This caused failures in use cases invoking the `nvidia-container-cli` directly or when an older `nvidia-container-toolkit` version was used with a newer `nvidia-container-cli`.
+- Added additional logging to the `nvidia-container-cli`.
+- Fixed variable initialisation when updating the ldcache. This caused failures on Arch linux or other platforms where the `nvidia-container-cli` was built from source.
+
+### Included Packages
+
+The following packages are included:
+
+- `nvidia-container-toolkit 1.17.8`
+- `nvidia-container-toolkit-base 1.17.8`
+- `libnvidia-container-tools 1.17.8`
+- `libnvidia-container1 1.17.8`
+
+The following `container-toolkit` containers are included:
+
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.17.8-ubi8`
+- `nvcr.io/nvidia/k8s/container-toolkit:v1.17.8-ubuntu20.04` (also as `nvcr.io/nvidia/k8s/container-toolkit:v1.17.8`)
 
 ## NVIDIA Container Toolkit 1.17.7
```

Comment:
Do you think its necessary to update the known issue section of 1.17.7 with a recommendation to upgrade to using 1.17.8 to avoid the bug? possibly unnecessary, since we already mention in the current note that the issue is fixed in the next patch release